### PR TITLE
Add paper about automated program repair and DeepLearning

### PR DESCRIPTION
**Pull request by martinezmatias**
Signed-off-by: Matias Martinez <matias.sebastian.martinez@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,7 @@ A curated list of awesome research papers, datasets and software projects devote
 * <img src="badges/8-pages-gray.svg" alt="8-pages" align="top"> [Estimating defectiveness of source code: A predictive model using GitHub content](https://arxiv.org/abs/1803.07764) - Ritu Kapur, Balwinder Sodhi, 2018
 * <img src="badges/8-pages-gray.svg" alt="8-pages" align="top"> [Automated software vulnerability detection with machine learning](https://arxiv.org/abs/1803.04497) - Jacob A. Harer, Louis Y. Kim, Rebecca L. Russell, Onur Ozdemir, Leonard R. Kosta, Akshay Rangamani, Lei H. Hamilton, Gabriel I. Centeno, Jonathan R. Key, Paul M. Ellingwood, Marc W. McConley, Jeffrey M. Opper, Peter Chin, Tomo Lazovich, IWSPA 2018
 * <img src="badges/12-pages-gray.svg" alt="12-pages" align="top"> [To Type or Not to Type: Quantifying Detectable Bugs in JavaScript](http://earlbarr.com/publications/typestudy.pdf) - Zheng Gao, Christian Bird, Earl Barr, ICSE 2017.
+* <img src="badges/12-pages-gray.svg" alt="12-pages" align="top"> [Sorting and Transforming Program Repair Ingredients via Deep Learning Code Similarities](https://arxiv.org/abs/1707.04742) - Martin White, Michele Tufano, Matías Martínez, Martin Monperrus, Denys Poshyvanyk, 2018.
```
Comment (EgorBu) [inline README.md]: please fix the year to 2017.

**Review by EgorBu — CHANGES_REQUESTED** [review]
Thanks for PR. Small fix and it can be merged!

File: README.md (hunk 1 above)
Comment (martinezmatias) [reply README.md]: Perfect, done :) Thanks @EgorBu!

**Comment by EgorBu** [comment]
Please sign your commit!

**Comment by EgorBu** [comment]
LGTM!

**Pull request merged**